### PR TITLE
Pin versions & run integration tests in separate folder

### LINKED ISSUE #57: Pin versions of walkthrough plugins
Need to pin these versions:

https://github.com/infinitered/ignite-ir-next/blob/master/boilerplate.js#L161-L190

### PR DESCRIPTION
**Pull request by jamonholmgren**
Fixes #57 and #58.

### CODE REVIEW

**Comment by jamonholmgren** [comment]
This is failing due to an older Ignite CLI being used in the tests, from what I can tell. Passes locally.

**Comment by GantMan** [comment]
I like it.  What's up with the tests though?  We should really get tests driven by source, and not by a server script.  That way tests can be actual barometers to success.

File: boilerplate.js [hunk 1]
```diff
@@ -162,29 +162,29 @@ async function install (context) {
 
     // now run install of Ignite Plugins
     if (answers['dev-screens'] === 'Yes') {
-      await system.spawn(`ignite add dev-screens ${debugFlag}`, {
+      await system.spawn(`ignite add dev-screens@"<2.1.0" ${debugFlag}`, {
```
Comment (skellock) [inline boilerplate.js]: can we make this 2.x.x?

File: boilerplate.js [hunk 2]
```diff
@@ -162,29 +162,29 @@ async function install (context) {
...
         stdio: 'inherit'
       })
     }
 
     if (answers['vector-icons'] === 'react-native-vector-icons') {
-      await system.spawn(`ignite add vector-icons ${debugFlag}`, {
+      await system.spawn(`ignite add vector-icons@"~>0.2.1" ${debugFlag}`, {
```
Comment (skellock) [inline boilerplate.js]: could we bump this lib to 1.0.0 and this to 1.x.x?

File: boilerplate.js [hunk 3]
```diff
@@ -162,29 +162,29 @@ async function install (context) {
...
         stdio: 'inherit'
       })
     }
 
     if (answers['i18n'] === 'react-native-i18n') {
-      await system.spawn(`ignite add i18n ${debugFlag}`, { stdio: 'inherit' })
+      await system.spawn(`ignite add i18n@"~>0.1.1" ${debugFlag}`, { stdio: 'inherit' })
```
Comment (skellock) [inline boilerplate.js]: 1.x.x would be great

File: boilerplate.js [hunk 4]
```diff
@@ -162,29 +162,29 @@ async function install (context) {
...
     }
 
     if (answers['animatable'] === 'react-native-animatable') {
-      await system.spawn(`ignite add animatable ${debugFlag}`, {
+      await system.spawn(`ignite add animatable@"~>0.3.1" ${debugFlag}`, {
```
Comment (skellock) [inline boilerplate.js]: We should bump this guy too.  and go 1.x.x

File: boilerplate.js [hunk 5]
```diff
@@ -162,29 +162,29 @@ async function install (context) {
...
         stdio: 'inherit'
       })
     }
 
     if (parameters.options.lint !== 'false') {
-      await system.spawn(`ignite add standard ${debugFlag}`, {
+      await system.spawn(`ignite add standard@"~>0.0.1" ${debugFlag}`, {
```
Comment (skellock) [inline boilerplate.js]: If @robinheinze is good with the interface, we should bump this to 1.x.x

File: package.json [hunk 6]
```diff
@@ -47,7 +47,8 @@
     "jest": "^19.0.2",
     "np": "^2.12.0",
     "sinon": "^1.17.7",
-    "standard": "^8.6.0"
+    "standard": "^8.6.0",
```
Comment (skellock) [inline package.json]: could we go 10.x.x? <3

**Comment by jamonholmgren** [comment]
@skellock It's the same as `~` as far as I can tell ... old ruby habit.

**Pull request merged**